src/ashell/shell-state.c

```diff
@@ -655,30 +652,20 @@ void ashell_run_boot_cfg()
 {
     fs_file_t *file;
     size_t count;
-
     file = fs_open_alloc("boot.cfg", "r");
-
     // Failed to open boot.cfg
     if (!file) {
         return;
     }
 
     ssize_t size = fs_size(file);
-    if (size == 0) {
-        printk("Empty boot.cfg file");
-        fs_close_alloc(file);
-        return;
-    }
-    else if (size > 12) {
-        fs_close_alloc(file);
-        return;
+    if (size > 0 && size <= 12) {
```

Comment:
This prints won't be seen because the uart isn't up yet, so no reason to keep them.